main.tf

```diff
@@ -5,21 +5,28 @@ locals {
   })
 }
 
-
-provider "google" {
-  project = var.project_id
-  region  = var.region
-}
-
-module "gke" {
-  source = "./modules/gke"
+module "networking" {
```

Comment:
Should we add the `data "google_client_config" ...` (from providers.tf) to `main.tf`?

Reply:
Seems like we always had it there and it was just duplicated in the prociders.tf file as well:

https://github.com/MaterializeInc/terraform-google-materialize/blob/c4eb8062bafaf917993fda0c6f32ac5c4d6957db/main.tf#L170